### PR TITLE
vimPlugins.trim-nvim: init at 2022-06-16

### PR DESCRIPTION
**Pull request by ivankovnatsky**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ivankovnatsky** [comment]
Result of `nixpkgs-review pr 187295` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>gnvim</li>
    <li>nvimpager</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>neovim</li>
    <li>neovim-qt</li>
    <li>spacevim</li>
    <li>vimPlugins.trim-nvim</li>
  </ul>
</details>

File: pkgs/applications/editors/vim/plugins/vim-plugin-names [hunk 1]
```diff
@@ -643,6 +643,7 @@ https://github.com/folke/tokyonight.nvim/,,
 https://github.com/markonm/traces.vim/,,
 https://github.com/tjdevries/train.nvim/,,
 https://github.com/tremor-rs/tremor-vim/,,
+https://github.com/cappyzawa/trim.nvim/,HEAD,
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/vim/plugins/vim-plugin-names:646]: Why do we need head here?

Comment (ivankovnatsky) [reply pkgs/applications/editors/vim/plugins/vim-plugin-names:646]: Hm, not sure, that's what the script added by itself, let me remove it.

Comment (ivankovnatsky) [inline pkgs/applications/editors/vim/plugins/vim-plugin-names:646]:
```suggestion
https://github.com/cappyzawa/trim.nvim/,,
```

**Pull request merged**